### ISSUE TITLE
remove the pages that are not needed in the webinterface (all the modules that we have not developed yet)

### ISSUE THREAD

**Issue by mlocardpaulet** [Proteobench/ProteoBench#147]
I don't really know how to proceed. I suspect that removing the .py files altogether is a bit extreme... But we should not have these empty pages present in the webinterface.

**Comment by scaramonche** [comment]
From what I could find out, I would suggest to use the `st_pages` package and a `pages.toml` file that could look like this:
```[[pages]]
path = "Home.py"
name = "Home"
icon = "🏠"

[[pages]]
path = "pages/DDA_Quant.py"
name = "DDA Quant Ion Level"
```

**Comment by scaramonche** [comment]
See branch `module_renaming`. To test it, install `st_pages` by running `pip install st-pages`.

**Comment by RobbinBouwmeester** [comment]
Dependencies can be added to (see dependencies header in the toml):

https://github.com/Proteobench/ProteoBench/blob/main/pyproject.toml

**Comment by scaramonche** [comment]
Fixed in pull request #156.
